src/nodeProvider.ts

```diff
@@ -42,7 +42,8 @@ export class DidactNodeProvider implements vscode.TreeDataProvider<TreeNode> {
         } else {
             // assume it's a category
             if (element.label && !(element instanceof TutorialNode)) {
-                return this.processTutorialsForCategory(element.label);
+				const tutorialCategory = retrieveTreeItemName(element);
+                return this.processTutorialsForCategory(tutorialCategory);
```

Comment:
seems this file contains a mix of space and tabs (but wasn't introduced by this PR